website/web/__init__.py

```diff
@@ -962,10 +962,14 @@ def urls_rendered_page(tree_uuid: str) -> WerkzeugResponse | str | Response:
 
 @app.route('/tree/<string:tree_uuid>/hashlookup', methods=['GET'])
 def hashlookup(tree_uuid: str) -> str | WerkzeugResponse | Response:
-    merged, total_ressources = lookyloo.merge_hashlookup_tree(tree_uuid)
-    # We only want unique URLs for the template
-    for sha1, entries in merged.items():
-        entries['nodes'] = {node.name for node in entries['nodes']}
+    try:
+        merged, total_ressources = lookyloo.merge_hashlookup_tree(tree_uuid)
+        # We only want unique URLs for the template
+        for sha1, entries in merged.items():
+            entries['nodes'] = {node.name for node in entries['nodes']}
+    except Exception:  # error or module not enabled
+        merged = []
```

Comment:
Now that I'm thinking about it, no need to render the `hashlookup.html` template there, you can simply render the tree flash an error message like that:

https://github.com/Lookyloo/lookyloo/blob/b6ff78f84985a078d2750716397bf119494dc1eb/website/web/__init__.py#L958-L962